frame/contracts/src/exec.rs

```diff
@@ -315,9 +315,9 @@ pub struct Stack<'a, T: Config, E> {
 	timestamp: MomentOf<T>,
 	/// The block number at the time of call stack instantiation.
 	block_number: T::BlockNumber,
-	/// The account counter is cached here when accessed. It is written back when the call stack
+	/// The trie_seed is cached here when accessed. It is written back when the call stack
 	/// finishes executing.
-	account_counter: Option<u64>,
+	trie_seed: Option<u64>,
```

Comment:
In the PR description you talk about `trie ids`, but here you're using `trie_seed`. From what I understand they're the same thing, so shouldn't all of these be `trie_id`'s?

Reply:
No. The `trie_seed` is a `u64` counter to make the trie_id unique for different instances of the same contract (if you terminate and recreate it).
```
trie_id = hash(contract_addr ++ trie_seed);
```

Reply:
So maybe to bike shed a bit, wouldn't something like `trie_nonce` be a better name in this case?

Reply:
Probably. Yes. I think this is a better name.